docs/source/general/contributing.rst

```diff
@@ -0,0 +1,14 @@
+Contributing
+------------
```

Comment:
use `=` signs for the main headers in order to be consistent with the other pages